src/coord/src/catalog/migrate.rs

```diff
@@ -122,6 +122,24 @@ fn ast_insert_default_confluent_wire_format_0_7_1(
                 })
             }
         }
+        Statement::CreateSource(CreateSourceStatement {
+            format:
+                CreateSourceFormat::Bare(Format::Avro(AvroSchema::Csr {
+                    csr_connector:
+                        CsrConnector {
+                            ref mut with_options,
```

Comment:
This is maybe excessively minor, but since the rhs of these match statements are the same, maybe just use a `|` on the lhs?

Reply:
They are similar, but not exactly the same! Unfortunately `AvroSchema::InlineSchema` uses `WithOption`s, but `AvroSchema::Csr` uses `SqlOption`s. (This was true before the refactor and just carried over).

Reply:
No! I did the same thing at first. I'm curious why we have both, and will look more closely in a follow up.